ui/jobs/jobs_config.ts

```diff
@@ -213,10 +213,8 @@ UserConfig.registerOptions('jobs', {
     {
       id: 'NotifyExpiredProcsInCombat',
       name: {
-        en:
-          'Flash procs boxes of inactive dots/etc. up to n times while in combat. (-1: disabled, 0: infinite)',
-        de:
-          'Dot/etc. boxen blinken bis zu n mal wenn im Kampf und dot ist nicht aktiv. (-1: deaktiviert, 0: ohne Limit)',
+        en: 'Flash procs boxes of inactive dots/etc. up to n times while in combat. (-1: disabled, 0: infinite)',
```

Comment:
A jumps of lines seem me abnormal, removing them, it's correct?

Reply:
This line is too long, so it requires to split into two lines.
Yeah, I think it is abnormal too. But the rule didn't expose more configurable settings for different situations.
So we have to keep it remains as before. So may you revert these two lines (en, de)?

Reply:
Ty for your explanation.